services/worker/src/worker/job_runners/dataset_size.py

```diff
@@ -107,7 +97,7 @@ def compute_sizes_response(dataset: str) -> SizesResponse:
         response = get_response(kind="/parquet-and-dataset-info", dataset=dataset)
```

Comment:
this step depends on `/parquet-and-dataset-info` but we don't express it directly in the processing graph:
- in the current model, a step only has one parent
- the parent `config-size` is the child of `/parquet-and-dataset-info`

To take into account if we change the model one day.